models/issue_tracker.go

```diff
@@ -0,0 +1,29 @@
+package models
+
+import (
+	"fmt"
+)
+
+// Configuration describes authentication model for the issue tracker.
+type Configuration struct {
+	ApiKey   string
+	Token    string
+	UserName string
+}
+
+// Issue gives information of every single issue of the issue trackers.
+type Issue struct {
+	Title       string
+	Description string
+}
+
+func PrintIssue(issue Issue) {
+	fmt.Println("title: ", issue.Title)
+	fmt.Println("issue: ", issue.Description)
+	fmt.Println("")
+}
+
+// Interface to Fetch data from issue trackers.
+type IssueProvider interface {
+	FetchData(chan string) (chan Issue, chan error)
```

Comment:
Sorry! We should actually make the "chan string" argument a "ctx contex.Contex"; https://godoc.org/golang.org/x/net/context#Context :)